modules/store/src/meta-reducers/immutability_reducer.ts

```diff
@@ -20,8 +20,7 @@ function freeze(target: any) {
     if (
       hasOwnProperty(target, prop) && targetIsFunction
         ? prop !== 'caller' && prop !== 'callee' && prop !== 'arguments'
-        : true &&
-          (isObjectLike(propValue) || isFunction(propValue)) &&
+        : (isObjectLike(propValue) || isFunction(propValue)) &&
```

Comment:
Seems like prettier in combination with myself messed this up, it should be:


```
function freeze(target: any) {
  Object.freeze(target);

  const targetIsFunction = isFunction(target);

  Object.getOwnPropertyNames(target).forEach(prop => {
    const propValue = target[prop];
    if (
      hasOwnProperty(target, prop) &&
      (targetIsFunction
        ? prop !== 'caller' && prop !== 'callee' && prop !== 'arguments'
        : true) &&
      (isObjectLike(propValue) || isFunction(propValue)) &&
      !Object.isFrozen(propValue)
    ) {
      freeze(propValue);
    }
  });

  return target;
```

Reference: [deep-freeze-strict](https://github.com/jsdf/deep-freeze/blob/master/index.js)

Reply:
Done.